### PR TITLE
Nikos/5931/coverage web3 abi

### LINKED ISSUE #5931: Coverage Web3 ABI
push unit tests coverage >= 80% of web3 abi https://app.codecov.io/gh/web3/web3.js/tree/4.x/packages

### PR DESCRIPTION
**Pull request by nikoulai**
## Description

web3-eth-abi coverage 

closes #5931

### CODE REVIEW

File: packages/web3-eth-abi/src/api/parameters_api.ts [hunk 1]
```diff
@@ -21,33 +21,6 @@ import { HexString, AbiInput, DecodedParams } from 'web3-types';
 import ethersAbiCoder from '../ethers_abi_coder';
 import { formatParam, isAbiFragment, mapTypes, modifyParams } from '../utils';
 
-/**
- * Helper function to format the decoded object
- */
-const formatDecodedObject = (
```
Comment (jdevcs) [inline packages/web3-eth-abi/src/api/parameters_api.ts:27]: Its not used in 4.x currently also we dnt have it in 1.x, but `formatDecodedObject` was part of last few 4.x releases so it should be added in changelog.

**Review by jdevcs — APPROVED** [review]
lgtm, just update changelog for `formatDecodedObject` . Thanks

**Review by avkos — APPROVED** [review]

File: packages/web3-eth-abi/src/api/parameters_api.ts (hunk 1 above)
Comment (Spacesai1or) [reply packages/web3-eth-abi/src/api/parameters_api.ts:27]: @luu-alex You added this function in #4844, was there an intended use for it?

**Review by Spacesai1or — APPROVED** [review]

File: packages/web3-eth-abi/src/api/parameters_api.ts (hunk 1 above)
Comment (luu-alex) [reply packages/web3-eth-abi/src/api/parameters_api.ts:27]: I think its a helper function I made that I never ended up using, we could get rid of it

Comment (nikoulai) [reply packages/web3-eth-abi/src/api/parameters_api.ts:27]: mentioned in changelog

**Pull request merged**